docs/source/reference/modding/settings/water.rst

```diff
@@ -58,17 +58,21 @@ This setting has no effect if the shader setting is false.
 
 This setting can be toggled with the 'Refraction' button in the Water tab of the Video panel of the Options menu.
 
-reflect actors
+reflection detail
 --------------
 
-:Type:		boolean
-:Range:		True/False
-:Default:	False
+:Type:		integer
+:Range:		0, 1, 2, 3
+:Default:	1
 
-This setting controls whether or not NPCs and creatures are drawn in water reflections.
-Setting this to true will enable actors in reflections and increase realism with a likely decrease in performance.
+Controls which object should be rendered in the water reflections in exteriors when the water shader is enabled.
+If the setting is 0, only terrain and sky will be reflected.
+If the setting is 1, also statics, activators and doors will be reflected.
+If the setting is 2, also items, containers and particles will be reflected.
+If the setting is 3, also actors will be reflected.
+In interiors all objects are reflected, player is always reflected too.
 
-This setting can be toggled with the 'Reflect actors' button in the Water tab of the Video panel of the Options menu.
+This setting can be configured via "Reflection shader detail" dropdown menu under Water section in the in-game settings menu.
```

Comment:
This setting controls the level of detail of exterior shader water reflections.
Level 0: only terrain and sky are reflected;
Level 1: statics, activators and doors are reflected;
Level 2: items, containers and particles are reflected;
Level 3: actors are reflected.

Player character is always reflected, and interior water always reflects everything.
This setting can be configured via 'Reflection shader detail' dropdown menu in the Water tab of the Video panel of the Options menu.

Reply:
Controls what kinds of things are rendered in water reflections.

0: only terrain and the sky are reflected (acts like level 1 in interiors)
1: statics, activators, and doors are also reflected
2: items, containers, and particles are also reflected
3: actors are also reflected

The player is always reflected in third-person mode. (?)
This setting can be changed ingame with the "Reflection shader detail" dropdown under the Water tab of the Video panel in the Options menu.